pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -92,6 +92,8 @@ const (
 	baseEphemeralDiskDir = "/var/run/kubevirt-ephemeral-disks/"
 )
 
+const maxHotplugHostDevicesConcurrent = 1
```

Comment:
I suggest changing this const name to "maxConcurrentHostDevicesHotplug"

Reply:
Done